site-src/concepts/use-cases.md

```diff
@@ -0,0 +1,177 @@
+# Gateway API Use Cases
+
+The Gateway API covers a _very_ wide range of use cases (which is both a
+strength and a weakness!). This page is emphatically _not_ meant to be an
+exhaustive list of these use cases: rather, it is meant to provide some
+examples that can be helpful to demonstrate how the API can be used.
+
+In all cases, it's very important to bear in mind the [roles and personas]
+used in the Gateway API. The use cases presented here are deliberately
+described in terms of [Ana], [Charlie], and [Ian]: they are the ones for whom the
+API must be usable. (It's also important to remember that even though these
+roles might be filled by the same person, especially in smaller organizations,
+they all have distinct concerns that we need to consider separately.)
+
+[roles and personas]:/concepts/roles-and-personas
+[Ana]:/concepts/roles-and-personas#ana
+[Charlie]:/concepts/roles-and-personas#charlie
+[Ian]:/concepts/roles-and-personas#ian
+
+## The Use Cases
+
+- [Basic north/south use case](#basic-north-south-use-case)
+- [Multiple applications behind a single
+  Gateway](#multiple-applications-behind-a-single-gateway)
+- [Basic east/west use case](#basic-east-west-use-case)
+- [Gateway and mesh use case](#gateway-and-mesh-use-case)
+
+[role and personas]:/concepts/roles-and-personas
+
+## Basic [north/south] use case
+
+Ana has created a microservice application which she wants to run in
+Kubernetes. Her application will be used by clients outside the cluster, and
+while Ana has created the application, setting up the cluster is not in her
+wheelhouse.
+
+1. Ana goes to Charlie to ask them to set up a cluster. Ana tells Charlie that
+   her clients will expect her APIs to be available using URLs rooted at
+   `https://ana.application.com/`.
+
+2. Charlie goes to Ian and requests a cluster.
+
+3. Ian provisions a cluster running a gateway controller with a [GatewayClass]
+   resource named `basic-gateway-class`. The gateway controller manages XXX.
```

Comment:
Comment to remind us all that this one needs to be finished up.

Reply:
Oh this is just depressing, thanks for the catch! I went off and wrote the glossary definition, then forgot to update this. :man_facepalming: